Makefile

```diff
@@ -352,6 +352,16 @@ intel-runtime-ml-ubi9-python-3.11: base-ubi9-python-3.11
 jupyter-intel-ml-ubi9-python-3.11: base-ubi9-python-3.11
 	$(call image,$@,jupyter/intel/ml/ubi9-python-3.11,$<)
 
+# Build and push base-habana-python-3.11 image to the registry
+.PHONY: base-habana-python-3.11
```

Comment:
Yeah, the base image already contains a Habana version on it, on this case, 1.17.1. I will rename it so it will become clear what is already bundled on this package altogether.

Reply:
Added version of Habana to the Makefile calls.